slasher/service/data_update.go

```diff
@@ -37,3 +46,134 @@ func (s *Service) finalisedChangeUpdater() error {
 		}
 	}
 }
+
+func (s *Service) slasherOldAtetstationFeeder() error {
```

Comment:
This function does a lot of things (at least 5 big things). Can we split up into smaller helper functions that are well-documented and piece it together that way?